content/en/docs/concepts/storage/volumes.md

```diff
@@ -1316,57 +1283,58 @@ deployments steps for each Kubernetes release and a compatibility matrix.
 {{< /note >}}
 
 Once a CSI compatible volume driver is deployed on a Kubernetes cluster, users
-may use the `csi` volume type to attach, mount, etc. the volumes exposed by the
+may use the `csi` volume type to attach or mount the volumes exposed by the
 CSI driver.
 
-A `csi` volume can be used in a pod in three different ways:
-- through a reference to a [`persistentVolumeClaim`](#persistentvolumeclaim)
-- with a [generic ephemeral volume](/docs/concepts/storage/ephemeral-volumes/#generic-ephemeral-volume) (alpha feature)
-- with a [CSI ephemeral volume](/docs/concepts/storage/ephemeral-volumes/#csi-ephemeral-volume) if the driver
-  supports that (beta feature)
+A `csi` volume can be used in a `Pod` in three different ways:
+
+* through a reference to a [PersistentVolumeClaim](#persistentvolumeclaim)
+* with a [generic ephemeral volume](/docs/concepts/storage/ephemeral-volumes/#generic-ephemeral-volume) (Alpha feature)
```

Comment:
I wouldn't capitalize Alpha. It's an adjective, the way I conceive it, so just as you wouldn't capitalize “new”, “deprecated” or “upcoming” I wouldn't capitalize this one.

Reply:
Yes, I guess so. I'd rather remove it.